nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -642,6 +712,44 @@ public Socket createSocket(InetAddress inetAddress, int i, InetAddress inetAddre
         return sftpClient;
     }
 
+    void updateConfigAlgorithms(Config config) {
+        if(ctx.getProperty(CIPHERS_ALLOWED).isSet()) {
+            Set<String> allowedCiphers = Arrays.stream(ctx.getProperty(CIPHERS_ALLOWED).evaluateAttributeExpressions().getValue().split(","))
+                    .map(String::trim)
+                    .collect(Collectors.toSet());
```

Comment:
To reduce duplication and ensure the same behavior, what do you think about refactoring similar property processing behavior to a shared method?  Something along the following lines:
```
private Set<String> getCommaSeparatedValues(final PropertyValue propertyValue) {
    Set<String> values = Collections.emptySet();
    if (propertyValue.isSet()) {
        values = Arrays.stream(propertyValue.evaluateAttributeExpressions().getValue().split(","))
        .map(String::trim)
        .collect(Collectors.toSet());
    }
    return values;
}
```
It may also be possible to create a shared method for collected allowed values, but at least creating a shared method for parsing property values would be helpful.

Reply:
I think that's unnecessary. The stream api already reduces boiler plate and I'd need to either check if each property was set or if the returned set was empty prior to using it to set a value.

If you were going to do something along these lines, I think a better use of your time would be to add some CSV capabilities to the PropertyValue class itself, allong with the StandardValidators.

Reply:
Thanks for the reply, it is fairly succinct as it stands, and I agree that it would probably be helpful to consider implementing a standardized approach for handling multiple values in a property.